dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,15 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        if (ident.spelling == "assert")
```

Comment:
> Does libclang expose such information? It is not well-defined in C in general because of copy-paste nature of headers

Yes, at least for regular non-macro symbols. It's possible to get the location/path of a cursor [1], not sure if macros are represented as cursors. It's used extensively by the include handler.

[1] https://github.com/jacob-carlborg/dstep/blob/master/clang/Cursor.d#L68